.github/workflows/deploy.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
       - name: Compile IGs
         run: yarn run compile-igs
       - name: Install serverless
-        run: npm install -g serverless
+        run: npm install -g serverless@2.64.1
```

Comment:
I wasn't sure how to not do this for the java serverless file as there isn't scripts